### PR TITLE
 permissions.yml 

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -595,6 +595,35 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+ terminator:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: improper spacing for terminator: node

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -595,6 +595,35 @@ groups:
...
+      survival:
+        permissions:
+        - ch.alias.jump0
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: set at jump 3 unless you have good proof of how high he can jump

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -595,6 +595,35 @@ groups:
...
+        - ch.alias.run2
+        - ch.alias.haste0
+        - ch.alias.haste1
+        - ch.alias.haste2
+        - ch.alias.haste3
+        - ch.alias.haste4
+        - ch.alias.haste5
+        - ch.alias.haste6
+        - ch.alias.haste7
+        - ch.alias.haste8
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Set as haste 5 unless you have extraordinary proof of his strength

Comment (adambash) [inline PermissionsEx/permissions.yml]: https://video.search.yahoo.com/yhs/search?fr=yhs-omr-001&hsimp=yhs-001&hspart=omr&p=the+terminator+beating+up+bill+paxton#id=1&vid=bda2c7df659042b251ae8ef4a2445911&action=click

Comment (adambash) [inline PermissionsEx/permissions.yml]: proof

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Thank you for the proof, but I have to say, It's not very impressive at all. I don't think that's nearly enough for haste 8. still request haste 5

Comment (adambash) [inline PermissionsEx/permissions.yml]: ok

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+ terminatorgenisys:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: spaced incorrectly.
is there a picture of the terminator genisys model? i can't find one on google

Comment (adambash) [inline PermissionsEx/permissions.yml]: https://www.google.com/search?q=Terminator+Genisys&espv=2&biw=1280&bih=640&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi3qoOxka7RAhVIiVQKHX8fD20Q_AUIBigB#imgrc=_WVLFmITF7jC2M%3A

**Comment by malonnnn** [comment]
File should be named permissions.yml
not
permissions.yml terminatorgenisys

File: PermissionsEx/permissions.yml terminatorgenisys [hunk 5]
```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+terminatorgenisys:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml terminatorgenisys]: I can't find any info on terminator genisys model, please show a link

Comment (adambash) [inline PermissionsEx/permissions.yml terminatorgenisys]: https://video.search.yahoo.com/yhs/search?fr=yhs-omr-001&hsimp=yhs-001&hspart=omr&p=the+terminator+beating+up+bill+paxton#id=1&vid=bda2c7df659042b251ae8ef4a2445911&action=click

Comment (malonnnn) [inline PermissionsEx/permissions.yml terminatorgenisys]: that looks like a t-800 model

Comment (adambash) [inline PermissionsEx/permissions.yml terminatorgenisys]: o

File: PermissionsEx/permissions.yml [hunk 6]
```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+terminator-t-800:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: no dashes allowed

File: PermissionsEx/permissions.yml [hunk 7]
```diff
@@ -595,6 +595,30 @@ groups:
...
+    options:
+      prefix: '&8T&7er&7m&8i&7n&4a&8t&7o&8r-&4T&8-8&700'
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: should be
terminator(t-800)

File: PermissionsEx/permissions.yml [hunk 8]
```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+terminatort-800:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: wrong spacing
dashes not allowed

File: PermissionsEx/permissions.yml [hunk 9]
```diff
@@ -595,6 +595,30 @@ groups:
...
+    options:
+      prefix: '&8T&7er&7m&8i&7n&4a&8t&7o&8r-&4T&8-8&700'
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this doesnt match the format of the .ms file

File: PermissionsEx/permissions.yml [hunk 10]
```diff
@@ -595,6 +595,30 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
+terminatort800:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is still spaced incorrectly

**Pull request closed without merging**